### ISSUE TITLE
Disingenuous naming

### ISSUE THREAD

**Issue by pmanole-volvocars** [daotl/ts-to-effect-schema#4]
I think it's a bit disingenuous to name your package `ts-to-effect-schema`, say you're targeting `@effect/schema` in your about header, then say you're targeting `Zod` in your README.md. Which is it? Did you do this just to snag a package name?

But it's your package, you name it what you want. I just think this puts a dark stain on your open source packages and your name.

**Comment by adrian-gierakowski** [comment]
The source was originally copied from a similar project which targeted zod. The author just didn’t bother to update the readme

**Comment by brokenthorn** [comment]
> The source was originally copied from a similar project which targeted zod. The author just didn’t bother to update the readme

OK. I guess. I still think though that a rename shouldn't have been that hard to do and it would help people find the package more easily as well by searching for what they want and getting what they expect as a result.
I certainly didn't expect to get a Zod library when I searched for Effect.

**Comment by mz-makerstudio** [comment]
Yes looks like readme is similar to this repo - https://github.com/fabien0102/ts-to-zod
